doc/user_manual/generated/internalRom.tex

```diff
@@ -2175,31 +2182,134 @@ \subsubsection{SyntheticHistory}
+      applies the hyperbolic tangent to the data and inverts by applying the
+      inverse hyperbolic tangent.
+      The \xmlNode{tanhtransformer} node recognizes the following parameters:
+        \begin{itemize}
+          \item \xmlAttr{target}: \xmlDesc{comma-separated strings, required}, 
+            indicates the variables for which this algorithm will be used for characterization.
+          \item \xmlAttr{seed}: \xmlDesc{integer, optional}, 
+            sets a seed for the underlying random number generator, if present.
+      \end{itemize}
+
+    \item \xmlNode{sigmoidtransformer}:
+      applies the sigmoid (expit) function to the data and inverts by applying
+      the logit function.
+      The \xmlNode{sigmoidtransformer} node recognizes the following parameters:
+        \begin{itemize}
+          \item \xmlAttr{target}: \xmlDesc{comma-separated strings, required}, 
+            indicates the variables for which this algorithm will be used for characterization.
+          \item \xmlAttr{seed}: \xmlDesc{integer, optional}, 
+            sets a seed for the underlying random number generator, if present.
+      \end{itemize}
+
+    \item \xmlNode{outtruncation}:
+      limits the data to either positive or negative values by "reflecting" the
+      out-of-range values back into the desired range.
+      The \xmlNode{outtruncation} node recognizes the following parameters:
+        \begin{itemize}
+          \item \xmlAttr{target}: \xmlDesc{comma-separated strings, required}, 
+            indicates the variables for which this algorithm will be used for characterization.
+          \item \xmlAttr{seed}: \xmlDesc{integer, optional}, 
+            sets a seed for the underlying random number generator, if present.
+          \item \xmlAttr{domain}: \xmlDesc{[positive, negative], required}, 
+            -- no description yet --
+      \end{itemize}
+
+    \item \xmlNode{quantiletransformer}:
+      transforms the data to fit a given distribution by mapping the data to     a uniform
+      distribution and then to the desired distribution.
+      The \xmlNode{quantiletransformer} node recognizes the following parameters:
+        \begin{itemize}
+          \item \xmlAttr{target}: \xmlDesc{comma-separated strings, required}, 
+            indicates the variables for which this algorithm will be used for characterization.
+          \item \xmlAttr{seed}: \xmlDesc{integer, optional}, 
+            sets a seed for the underlying random number generator, if present.
+          \item \xmlAttr{nQuantiles}: \xmlDesc{integer, optional}, 
+            number of quantiles to use in the transformation. If \xmlAttr{nQuantiles}
+            is greater than the number of data, then the number of data is used instead. \default{1000}
+          \item \xmlAttr{outputDistribution}: \xmlDesc{[normal, uniform], optional}, 
+            distribution to transform to. Must be either 'normal' or 'uniform'. \default{normal}
```

Comment:
did you check if this compiles as expected?

Reply:
I don't have a full regression test that looks at that, but in the testNormalization.py unit tests included in the PR, both options are tested. If you're talking about the actual docstring loading properly into the user manual, it seems to compile as expected.
![image](https://github.com/idaholab/raven/assets/23038884/dc79a88e-c896-444c-a985-2dbf15cecdc3)

Reply:
Though the single quotes are a little wonky, I suppose.

Reply:
LGTM.